bin/impl/download.sh

```diff
@@ -40,6 +40,76 @@ else
   wget -c -P $DOWNLOADS $APACHE_MIRROR/$ACCUMULO_PATH/$ACCUMULO_TARBALL
 fi
 
+if [ $SETUP_METRICS = "true" ]; then
+
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    BUILD_INFLUXDB=true
+    BUILD_GRAFANA=true
+  fi
+
+  BUILD=$DOWNLOADS/build
+  GOPATH=$BUILD/go
+  rm -rf $BUILD
+  mkdir -p $BUILD
+  mkdir -p $GOPATH
+  IF_DIR=influxdb-$INFLUXDB_VERSION
+  IF_PATH=$BUILD/$IF_DIR
+  GF_DIR=grafana-$GRAFANA_VERSION
+  GF_PATH=$BUILD/$GF_DIR
+
+  echo "Downloading InfluxDB tarball"
+  INFLUXDB_TARBALL=influxdb_"$INFLUXDB_VERSION"_x86_64.tar.gz
+  wget -c -P $DOWNLOADS https://s3.amazonaws.com/influxdb/$INFLUXDB_TARBALL
```

Comment:
Did you look into md5 hashes?   I suppose since its downloading over https its not too big of a concern.  I looked around on their websites and could not find any sort of hash or signature for grafana.   For influxdb I found md5 hashes embeded on their download page.